spec/menu_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'menu'
+
+describe Menu do 
+
+	subject(:menu) {described_class.new}
+
+	describe '#menu' do
+
+		it 'shows a list of dishes' do
+			expect(menu.list_dishes).to eq(Hash.new)
```

Comment:
Style/EmptyLiteral: Use hash literal `{}` instead of `Hash.new`. (https://github.com/bbatsov/ruby-style-guide#literal-array-hash)